libs/blocks/notification/notification.css

```diff
@@ -362,10 +364,6 @@
     inline-size: auto;
   }
 
-  .notification.ribbon .foreground.container .icon-area {
-    flex-shrink: 1;
-  }
```

Comment:
Can a notification be both a pill AND a ribbon at the same time? I'm just wondering because the removed style related to the latter, while the PR is trying to address the former and want to make sure we're not removing something that might have value for other use-cases.

Reply:
Good point. This came up after another authoring audit, so the hope was to combine that work with this work. I'll keep them separate and work with the reporter to get a ticket made and make a separate PR 👍